include/pokemon_3.h

```diff
@@ -3,13 +3,14 @@
 
 #include "pokemon.h"
 
+// size: 0x8
 typedef struct EvolveStatus
 {
-    u8 evoItem1;
-    u8 evoItem2;
-    u8 unk6;
-    u16 evolutionConditionStatus;
-    s16 targetEvolveSpecies;
+    /* 0x0 */ u8 evoItem1;
+    /* 0x1 */ u8 evoItem2;
+    /* 0x2 */ u8 wurmpleVal;
```

Comment:
What exactly is this name supposed to mean? Maybe a clarifying comment is needed.

Reply:
(if I had to guess, this is the flag for silcoon vs cascoon)